test/index.ts

```diff
@@ -13,10 +13,23 @@ export type Test = () => void;
 
 export { getCanisterActor } from './get_canister_actor';
 
+export interface RunTestsOptions {
```

Comment:
Why is this an `interface`? We usually use `type` and I believe we should avoid interfaces.

Reply:
done